### PR TITLE
add acho of chinese language

### PR DESCRIPTION
**Pull request by Yatoo2018**
translate the readme.md file to chinese

### CODE REVIEW

**Comment by edmorley** [comment]
@eliperelman what are your thoughts about this? It seems fine to me (only consideration perhaps whether to move the link down to the end of the introduction section etc)

**Comment by mc-zone** [comment]
I think maybe put translated docs to the webpack-chain repo is better for maintenance in future?

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ This documentation corresponds to v5 of webpack-chain. For previous versions, se
 _Note: while webpack-chain is utilized extensively in Neutrino, this package is
 completely standalone and can be used by any project._
 
+**[chinese docs（中文文档）](https://github.com/Yatoo2018/webpack-chain/tree/zh-cmn-Hans)**
```
Comment (mc-zone) [inline README.md]: a little wrong with the case:
```diff
- [chinese docs（中文文档）]
+ [Chinese docs (中文文档)]
``` 
also using half-width brackets might better :)

**Comment by edmorley** [comment]
> I think maybe put translated docs to the webpack-chain repo is better for maintenance in future?

Yeah I think this might be best. @Yatoo2018, would you mind updating the PR? :-)

**Comment by Yatoo2018** [comment]
> > I think maybe put translated docs to the webpack-chain repo is better for maintenance in future?
> 
> Yeah I think this might be best. @Yatoo2018, would you mind updating the PR? :-)

It 's OK.

**Pull request merged**